ballista/core/src/execution_plans/shuffle_reader.rs

```diff
@@ -184,7 +184,11 @@ impl ExecutionPlan for ShuffleReaderExec {
         // Sort partitions for evenly send fetching partition requests to avoid hot executors within one task
         let mut partition_locations: Vec<PartitionLocation> = partition_locations
             .into_values()
-            .flat_map(|ps| ps.into_iter().enumerate())
+            .flat_map(|ps| {
```

Comment:
We just need to test this and make sure it works correctly when all partitions are empty

Reply:
I plan to do it on staging